packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.txt

```diff
@@ -1,8 +1,9 @@
-verifyLoginCode-title = "Is this you signing in?"
+verifyLoginCode-title-2 = "Did you sign in to <%- serviceName %>?"
 
+verifyLoginCode-request-info = "Help us keep your account safe by approving you signed in on:"
 <%- include('/partials/userInfo/index.txt') %>
 
-verifyLoginCode-prompt-2 = "If yes, here is the confirmation code:"
+verifyLoginCode-prompt-3 = "If yes, here is your authorization code:"
 <%- code %>
 
 verifyLoginCode-expiry-notice = "It expires in 5 minutes."
```

Comment:
Can you please replace the `changePassword` partial below with the `automatedEmailNoAction`, so that it matches the fineprint in MJML? Thanks! :)

<img width="825" alt="Screen Shot 2022-08-15 at 12 08 47 PM" src="https://user-images.githubusercontent.com/28129806/184672355-71672ffa-7342-4571-bed6-cde2697ae2f8.png">